### PR TITLE
Feature/auth check

### PR DESCRIPTION
**Pull request by MatthiasWanner**
Add request in layout to check if user is authenticated

### CODE REVIEW

File: src/API/query-client.ts [hunk 1]
```diff
@@ -0,0 +1,11 @@
+import { QueryClient } from 'react-query';
+
+const queryClient = new QueryClient({
+  defaultOptions: {
+    queries: {
+      refetchOnWindowFocus: false,
+    },
+  },
+});
```
Comment (Karnak19) [inline src/API/query-client.ts]:
```js
const queryClient = new QueryClient({
  defaultOptions: {
    queries: {
      refetchOnWindowFocus: import.meta.env.PROD ? true : false,
    },
  },
});
```

File: src/views/Layout.tsx [hunk 2]
```diff
@@ -13,6 +16,30 @@ function Layout(): JSX.Element {
     'sm:flex flex-col bg-black w-full h-full rounded-xl text-white font-roboto justify-between shadow-mainShadow invisible sm:visible'
   );
 
+  const history = useHistory();
+
+  const { isLoading } = useQuery<{ message: string; user: User }>('userAuthenticated', () => auth.me(), {
+    onSuccess: (data) => {
+      console.log(data);
```
Comment (Karnak19) [inline src/views/Layout.tsx]: `console.log`

**Review by Karnak19 — CHANGES_REQUESTED** [review]
Check the environment on the queryClient so we keep refetch on window focus in production

File: src/API/query-client.ts (hunk 1 above)
Comment (MatthiasWanner) [reply src/API/query-client.ts]: ok it's corrected

File: src/views/Layout.tsx (hunk 2 above)
Comment (MatthiasWanner) [reply src/views/Layout.tsx]: Ok thanks. I deleted it

**Review by Chabelle78 — APPROVED** [review]

**Review by Karnak19 — APPROVED** [review]

**Pull request merged**